test/torchaudio_unittest/prototype/functional/torchscript_consistency_test_impl.py

```diff
@@ -26,7 +26,7 @@ def _assert_consistency(self, func, inputs, shape_only=False):
         self.assertEqual(ts_output, output)
 
     @nested_params(
-        [F.convolve, F.fftconvolve],
+        ["convolve", "fftconvolve"],
```

Comment:
Changing this so that the memory address of the function is not leaked into the generated test names.